components/shared/DropdownMobile.js

```diff
@@ -2,17 +2,10 @@
 
 import * as React from "react";
 import styled from "styled-components";
-import Text from "@kiwicom/nitro/lib/components/Text";
 
-type DropdownItem = {
-  id: number,
-  value: string,
-  label: string,
-};
-
-export const renderDropdownItem = ({ value, id, label }: DropdownItem) => (
-  <option value={value} key={id}>
-    <Text t={label} />
```

Comment:
Don't we need translations even here? Or are they managed in some other way now?

Reply:
No, because we translate the key before giving it here. I refactored it to have a string as a child because otherwise we get a warning that option should not contain components, only strings